core/navigation.js

```diff
@@ -20,10 +20,43 @@
 
 goog.provide('Blockly.Navigation');
 
+/**
+ * The current connection the cursor is on.
+ */
 Blockly.Navigation.connection = null;
 
+/**
+ * The current selected category.
+ */
 Blockly.Navigation.curCategory = null;
 
+/**
+ * The current selected block in flyout.
+ */
+Blockly.Navigation.flyoutBlock = null;
+
+/**
+ * The selected connection.
+ */
+Blockly.Navigation.markedConnect = null;
+
+/**
+ * State for the flyout.
```

Comment:
These comments are too vague. Should be something like:
"State indicating focus is currently in a flyout."